sklearn/ensemble/tests/test_stacking.py

```diff
@@ -0,0 +1,218 @@
+"""
+Testing for the stacking ensemble module (sklearn.ensemble.stacking).
+"""
+
+# Author: Caio Oliveira
+# License BSD 3 clause
+
+
+from copy import deepcopy
+import numpy as np
+from scipy.sparse import csr_matrix
+
+from sklearn.utils.mocking import CheckingClassifier
+from sklearn.utils.testing import (assert_equal, assert_array_equal,
+                                   assert_false)
+from sklearn.utils.testing import SkipTest
+from sklearn.ensemble import (StackingTransformer, make_stack_layer)
+from sklearn.linear_model import (RidgeClassifier, LinearRegression)
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.svm import LinearSVC, LinearSVR, SVC
+from sklearn import datasets
+from sklearn.model_selection import (ParameterGrid, StratifiedKFold)
+
+iris = datasets.load_iris()
+X, y = iris.data[:, 1:3], iris.target
+
+RANDOM_SEED = 8939
+
+META_ESTIMATOR_PARAMS = {'cv': [2, StratifiedKFold()],
+                         'n_jobs': [1, 2]}
+META_ESTIMATOR_FIT_PARAMS = [{}, {"sample_weight": np.ones(y.shape)}]
+
+
+def _check_estimator(estimator, **fit_params):
+    # checks that we can fit_transform to the data
+    Xt = estimator.fit_transform(X, y, **fit_params)
+
+    # checks that we get a column vector
+    assert_equal(Xt.ndim, 2)
+
+    # checks that `fit` is available
+    estimator.fit(X, y, **fit_params)
+
+    # checks that we can transform the data after it's fitted
+    Xt2 = estimator.transform(X)
+
+    # checks that transformed data is always a column vector
+    assert_equal(Xt2.ndim, 2)
+
+    # checks that the generated features is diferent from the original ones
```

Comment:
"original ones" is a bit strange here. Maybe just write "check that transform is different from fit_transform"